.changeset/shaggy-deers-destroy.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+Improves SSR performance for synchronous components by avoiding use of Promises.
```

Comment:
Just noting that this can totally sound a bit like the blog post here! H.Y.P.E.!